account_financial_report/report/journal_ledger_xlsx.py

```diff
@@ -200,12 +200,14 @@ def _generate_no_group_content(self, workbook, report):
             workbook, report, "Report", report.report_move_ids)
         self._generate_no_group_taxes_summary(workbook, report)
 
+    def _get_sheet_name(self, report_journal, pre=None, post=None):
```

Comment:
```suggestion
    def _get_sheet_name(self, report_journal, pre='', post=''):
```

so that you don't need the ` or ''` part